### PR TITLE
WIP: Bob/manifest5

### PR DESCRIPTION
**Pull request by apark263**
For review

### CODE REVIEW

File: loader/src/batch_iterator_async.cpp [hunk 1]
```diff
@@ -59,40 +47,30 @@ variable_buffer_array* batch_iterator_async::filler()
...
         }
         else
         {
-            move_count = number_needed; // Enough in the block to service this batch and more
+            move_count = remainder; // Enough in the block to service this batch and more
         }
 
-        move_src_to_dst(m_input_ptr, rc, move_count);
+        // swap records one at a time
+        m_input_ptr->move_to(*rc, move_count);
```
Comment (apark263) [inline loader/src/batch_iterator_async.cpp]: isn't this going to be problematic since we are ignoring `move_count`?

Comment (rkimballn1) [inline loader/src/batch_iterator_async.cpp]: this is move_to() implemented in buffer_batch.hpp. I need to delete the move_src_to_dst method as it should no longer be used.

Comment (apark263) [inline loader/src/batch_iterator_async.cpp]: ah, i missed that

**Comment by apark263** [comment]
merged separately

**Pull request closed without merging**